packages/Switch/src/Switch.js

```diff
@@ -38,8 +37,7 @@ function Switch(props) {
   const { a11yText, canPropagate, isChecked, isDisabled, size, onChange, ...moreProps } = props;
 
   return (
-    <RawButton
-      css={switchStyles}
+    <sc.switchStyles
```

Comment:
I think you missed this one... should be `<sc.switchWrapper>` – or, as I'd recommend, just `<sc.Switch>`.